js/packages/phoenix-client/src/experiments/runExperiment.ts

```diff
@@ -17,6 +17,23 @@ import { getDatasetBySelector } from "../utils/getDatasetBySelector";
 import { pluralize } from "../utils/pluralize";
 import { promisifyResult } from "../utils/promisifyResult";
 import { AnnotatorKind } from "../types/annotations";
+import { instrument } from "./instrument";
+import {
+  AttributeValue,
+  SpanStatusCode,
+  Tracer,
+  trace,
+} from "@opentelemetry/api";
+import {
+  MimeType,
+  OpenInferenceSpanKind,
+  SemanticConventions,
+} from "@arizeai/openinference-semantic-conventions";
+import { ensureString } from "../utils/ensureString";
+
+const INSTRUMENTATION = {
+  enabled: false,
+};
```

Comment:
i would probably re-factor to handle exits etc. and un-register. maybe have this actually just point to the provider so if the provider is set, we unregister